mail_tracking/models/mail_composer.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2019 Alexandre Díaz
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields, api
+
+
+class MailComposer(models.TransientModel):
+    _inherit = 'mail.compose.message'
+
+    hide_followers = fields.Boolean(string="Hide follower message",
+                                    default=False)
+
+    @api.multi
+    def send_mail(self, auto_commit=False):
+        """ This method marks as reviewed the message when using the 'Retry'
+            option in the mail_failed_message widget"""
```

Comment:
According to [PEP 257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings):

> Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description.

Example:

```python
        """Make compatible with mail_failed_message widget.

        Mark as reviewed when using 'Retry' option in that widget.
        """
```

:blush: